components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilderTest.java

```diff
@@ -157,14 +169,114 @@ public void setUp() throws Exception {
 
     }
 
-    @Test
-    public void testBuildIDToken() throws Exception {
+    @DataProvider(name = "TestBuildRequestIDToken")
+    public Object[][] buildRequestIDToken() throws Exception {
+        // second set of values
+        OAuth2AccessTokenReqDTO tokenReqDTO2 = new OAuth2AccessTokenReqDTO();
+        tokenReqDTO2.setTenantDomain(SUPER_TENANT_DOMAIN_NAME);
+        tokenReqDTO2.setClientId("dabfba9390aa423f8b04332794d83614");
```

Comment:
since this client ID value is used in multiple places, let's declare it as a constant and use it